index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+        <h1 id="title">Survey form</h1>
+        <p id="description">Please fill this form</p>
+        <form id="survey-form">
+           <div> <label id="name-label" for="name">Name</label>
+            <input id="name" placeholder="name" type="text" required>
+        </div>
+
+            
+        <div>
+            <label id="email-label" for="email">Email</label>
+            <input id="email" Placeholder="email" type="email" required>
+        </div>
+            
+        <div><label id="number-label" for="number">Number</label>
+            <input id="number" type="number" placeholder="number" min="10" max="99" required>
+        </div>
+
+            <div>
+                <label id="dropdown-label" for="drowpdown"></label>
+            <select id="dropdown"></select>
+            </div>
+                <div>
+            <p>Would you recommend Us to a friend?</p>
+            <label>
+                <input
+                  name="user-recommend"
+                  value="definitely"
+                  type="radio"
+                  class="input-radio"
+                  checked
+                />Definitely</label>
+                </div>
+
+               <div> <input
+                name="user-recommend"
+                value="maybe"
+                type="radio"
+                class="input-radio"
+              />Maybe</label>
+               </div>
+              <div>
+                <p>What is your favorite feature of US?</p>
+              <select id="most-like" name="mostLike" class="form-control" required>
+                <option disabled selected value>Select an option</option>
+                <option value="challenges">Challenges</option>
+                <option value="projects">Projects</option>
+                <option value="community">Community</option>
+                <option value="openSource">Open Source</option>
+              </select>
+              </div>
+              <div>
+              <textarea placeholder="Enter Your comments"></textarea>
```

Comment:
Please make sure that the <textarea> tag looks like the child element of its parent div. For now, it is looking like they are siblings!